walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -247,6 +262,11 @@ Verify that you followed each step in the procedure above.  If you are still hav
 To explore data sync features, you should run multiple instances of the showcase app using different browsers.
 For example, use the browser on your mobile device as well as using the browser on your laptop.
 
+To get the url of your app:
+
+. Navigate to the {openshift-host}/console/project/{walkthrough-namespace}/browse/routes[Routes page of the OpenShift console]
```

Comment:
can replace all this with 

Navigate to the link:{route-ionic-showcase-server-host}[Data Sync user interface]